### PR TITLE
Add generative NLG

### LINKED ISSUE #187: Add generative NLG

### PR DESCRIPTION
**Pull request by NoB0**
Fixes #187

### CODE REVIEW

File: usersimcrs/nlg/lm/nlg_generative_lm.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+              forced or not. Not used in this NLG.
+
+        Raises:
+            RuntimeError: If generation fails.
+
+        Returns:
+            Natural language utterance.
+        """
+        try:
+            str_dialogue_acts = self._stringify_dialogue_acts(dialogue_acts)
```
Comment (kbalog) [inline usersimcrs/nlg/lm/nlg_generative_lm.py:73]: Nit: `str_dialogue_acts` => `dialogue_acts_str`

File: usersimcrs/nlg/lm/nlg_generative_lm.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+            if annotations:
+                str_annotations = self._stringify_annotations(annotations)
```
Comment (kbalog) [inline usersimcrs/nlg/lm/nlg_generative_lm.py:76]: Nit: `str_annotations` => `annonations_str`

**Review by kbalog — APPROVED** [review]
LGTM

File: usersimcrs/nlg/lm/nlg_generative_lm.py (hunk 1 above)
Comment (NoB0) [reply usersimcrs/nlg/lm/nlg_generative_lm.py:73]: Done.

File: usersimcrs/nlg/lm/nlg_generative_lm.py (hunk 2 above)
Comment (NoB0) [reply usersimcrs/nlg/lm/nlg_generative_lm.py:76]: Done.

**Pull request merged**